src/qt/forms/createwalletdialog.ui

```diff
@@ -68,17 +71,30 @@
     <string>Encrypt Wallet</string>
    </property>
    <property name="checked">
-    <bool>true</bool>
+    <bool>false</bool>
+   </property>
+  </widget>
+  <widget class="QLabel" name="encrypt_wallet_label">
+   <property name="geometry">
+    <rect>
+     <x>20</x>
+     <y>90</y>
+     <width>130</width>
+     <height>21</height>
+    </rect>
+   </property>
+   <property name="text">
+    <string>Advanced options</string>
```

Comment:
you say that there are no new strings, but this looks like a new string. why is it not a new string?

Reply:
why is it not a heading like h1-6? seems like it is supposed to cover multiple items below?

Reply:
There's no modified strings, but indeed this is added. It will show in English for all users.

Reply:
I changed it to bold font, same as with Coin Selection in the send screen.

Reply:
Hmm... thats unfortunate. Would using "Options" (already translated) be an option?